GameFiles/_integration.gsc

```diff
@@ -706,13 +739,30 @@ UnhideImpl()
     self IPrintLnBold( "You are now ^5visible ^7to other players" );
 }
 
-AlertImpl( data )
+AlertImpl( event, data )
 {
-    self thread maps\mp\gametypes\_hud_message::oldNotifyMessage( data["alertType"], data["message"], "compass_waypoint_target", ( 1, 0, 0 ), "ui_mp_nukebomb_timer", 7.5 );
+    if (level.eventBus.gamename == "IW4") {
+        self thread maps\mp\gametypes\_hud_message::oldNotifyMessage( data["alertType"], data["message"], "compass_waypoint_target", ( 1, 0, 0 ), "ui_mp_nukebomb_timer", 7.5 );
+    }
+    if (level.eventBus.gamename == "IW5") { //IW5's notification are a bit different...
+        self thread maps\mp\gametypes\_hud_message::oldNotifyMessage( data["alertType"], data["message"], undefined, ( 1, 0, 0 ), "ui_mp_nukebomb_timer", 7.5 );
```

Comment:
it's nice having the ability to switch this way. If this pattern becomes more common we can even do a mapping from game name to function or something like that